vmck/backends/docker.py

```diff
@@ -20,12 +21,15 @@ def task_group(self, job, options):
             'services': services(job),
         }
 
+        tasks = [docker_vm_task]
+
+        if options.get('manager', False):
```

Comment:
Q: This line is added in the case you do not want your job to be submitted?

Reply:
That's for when you use `vmck` with an external vagrant that you control. `manager` is for when you want a self-contained task on the queue, like we do with `acs-interface`.